src/ResourceManager/Network/Commands.Network/Commands.Network.Netcore.csproj

```diff
@@ -68,4 +68,8 @@
   <ItemGroup>
     <Content Include="help\**\*" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\KeyVault\Commands.KeyVault\Commands.KeyVault.Netcore.csproj" />
```

Comment:
@akshaysngupta please remove this reference -- as mentioned before, we cannot introduce a coupling between modules